ee/reconciliation/internal/storage/policy.go

```diff
@@ -0,0 +1,52 @@
+package storage
+
+import (
+	"context"
+
+	"github.com/formancehq/reconciliation/internal/models"
+	"github.com/google/uuid"
+)
+
+func (s *Storage) CreatePolicy(ctx context.Context, policy *models.Policy) error {
+	var id uuid.UUID
```

Comment:
I'm not a huge fan of letting the database generate the id.
It is a simple case where the model can do a plain uuid.NewString()